main.py

```diff
@@ -74,6 +84,79 @@ def player_set_volume():
     return jsonify({'message': 'No volume parameter'}), 400
 
 
+@app.route('/v1/equalizer_info', methods=['GET'])
```

Comment:
On second thought, it probably makes more sense for this one in particular to just be `GET /v1/player/equalizer`.